### ISSUE TITLE
Grafana -> Prother Especifications

### ISSUE THREAD

**Issue by matheuscamarques** [matheuscamarques/poc_shopping_cart#4]
Can you create a specification on how to use Grafana with Prometheus? I tried to configure it here and ended up not being able to get the Prometheus metrics in Grafana

@leandroaraujodeveloper

**Comment by leandroaraujodeveloper** [comment]
Yes I do, but I think we need to create more metrics because it is possible o see only default metrics there at this moment.

**Comment by matheuscamarques** [comment]
I would like a step-by-step list of how to raise the project and see some metrics that are already defined
